scripts/cuvslam_kpi_report.py

```
@@ -0,0 +1,447 @@
+    parser.add_argument('-j', '--out_kpi_json', type=str, help='full path to output KPI json')
+    parser.add_argument('-d', '--run_id', type=str, help='run id label for logs/report', default="")
+    parser.add_argument('-k', '--prev_kpi', type=str, help='path to previous KPI json file for diff calculation', default="")
+    parser.add_argument('-b', '--baseline_ranges', type=str, help='path to committed KPI baseline-ranges json for a soft drift check', default="")
+    args = parser.parse_args()
+
+    kpis_dict = {}
+
+    required_metrics = ["ATE", "ARE", "Kabsch", "TrackingLosts", "FPS"]
+
+    if not os.path.exists(args.stat_folder):
+        raise ValueError(f'Stat folder does not exist: {args.stat_folder}')
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)